tests/test_conditional_steps.py

```diff
@@ -98,47 +95,54 @@ def _build_step_command(files_to_create, exit_code) -> List[str]:
         return ["bash", "-c", ";".join(commands)]
 
     def _write_config_file(self, steps_info) -> None:
-        true_branch_step: str = f"Configuration([Step(**{str(steps_info.true_branch_step)})])" \
-            if steps_info.true_branch_step else "None"
-        false_branch_step: str = f"Configuration([Step(**{str(steps_info.false_branch_step)})])" \
-            if steps_info.false_branch_step else "None"
+        true_branch_config = self._build_branch_configuration_string(steps_info.true_branch_steps)
+        false_branch_config = self._build_branch_configuration_string(steps_info.false_branch_steps)
         config_lines: List[str] = [
             "from universum.configuration_support import Configuration, Step",
-            f"true_branch_step = {true_branch_step}",
-            f"false_branch_step = {false_branch_step}",
+            f"true_branch_config = {true_branch_config}",
+            f"false_branch_config = {false_branch_config}",
             f"conditional_step = Step(**{str(steps_info.conditional_step)})",
 
             # `true/false_branch_steps` should be Python objects from this script
             "conditional_step.is_conditional = True",
-            "conditional_step.if_succeeded = true_branch_step",
-            "conditional_step.if_failed = false_branch_step",
+            "conditional_step.if_succeeded = true_branch_config",
+            "conditional_step.if_failed = false_branch_config",
 
             "configs = Configuration([conditional_step])"
         ]
         config: str = "\n".join(config_lines)
 
         self.configs_file.write_text(config, "utf-8")
 
+    @staticmethod
+    def _build_branch_configuration_string(steps: Optional[List[Step]]) -> str:
+        if not steps:
+            return "None"
```

Comment:
Is this string intended to return value that looks alike to actual `None`?

Reply:
Yes, as this value is inserted in a Python config file in the `_write_config_file()` method and then interpreted by Universum.